drivers/gles2/shaders/cubemap_filter.glsl

```diff
@@ -40,8 +40,8 @@ void main() {
 #endif
 
 #if !defined(GL_EXT_shader_texture_lod) && !defined(GL_ARB_shader_texture_lod)
-#define texture2DLod(img, coord, lod) texture2D(img, coord, lod)
-#define textureCubeLod(img, coord, lod) textureCube(img, coord, lod)
+#define texture2DLod(img, coord, lod) texture2D(img, coord)
+#define textureCubeLod(img, coord, lod) textureCube(img, coord)
```

Comment:
These changes are unnecessary, so it is best not to make them.

Reply:
Reverted in 4b95b18.